Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -172,6 +179,16 @@ struct TimelineStepPreviews: PreviewProvider {
                 style: .status(.critical),
                 content: "We’ve assigned your request to one of our agents."
             )
+
+            TimelineStep(
+                "Requested",
+                sublabel: "3rd May 14:04",
+                style: .status(.info)) {
```

Comment:
Formatting: closing bracket.

Reply:
updated, thanks.

Reply:
We could probably add a SwiftPM plugin that invokes SwiftFormat to automate this 🤔